lib/pkgcloud/azure-v2/compute/client/servers.js

```diff
@@ -27,7 +27,9 @@ function getVersion(callback) {
  */
 function getLimits(callback) {
   return errs.handle(
-    errs.create({ message: 'Azure\'s API is not rate limited' }),
+    errs.create({
+      message: 'Azure\'s API is not rate limited'
+    }),
```

Comment:
General remark - I guess you are "beautifying" the code - but this should be done separarately from major changes since I can't tell what are the real changes and what are "white spaces"